_client/livethere-app/src/components/search/searchBar.js

```diff
@@ -94,15 +146,16 @@ const Search = (props) => {
             placeholder="Search your university/college"
             onChange={handleInputChange}
             onKeyUp={handleKeyPress}
+            onClick={()=>{ if (keyword !== '') setShowDropbox(true)}}
```

Comment:
can you just make sure codes are in similar format? You can get jsx formatted or react formatted to do them. We have different spaces here and there.

Reply:
()=>{ if } to be () => { if }

Reply:
just "believe"